server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -566,66 +529,15 @@ export class AssetService {
     return { filepath: asset.resizePath, contentType: 'image/jpeg' };
   }
 
-  private async streamFile(filepath: string, res: Res, headers: Record<string, string>, contentType?: string | null) {
-    await fs.access(filepath, constants.R_OK);
-    const { size, mtimeNs } = await fs.stat(filepath, { bigint: true });
-
+  async streamFile(res: Res, filepath: string, contentType?: string | null) {
     if (contentType) {
-      res.header('Content-Type', contentType);
+      res.contentType(contentType);
     }
-
-    const range = this.setResRange(res, headers, Number(size));
-
-    // etag
-    const etag = `W/"${size}-${mtimeNs}"`;
-    res.setHeader('ETag', etag);
-    if (etag === headers['if-none-match']) {
-      res.status(304);
-      return;
-    }
-
-    const stream = createReadStream(filepath, range);
-    return await pipeline(stream, res).catch((err) => {
-      if (err.code !== 'ERR_STREAM_PREMATURE_CLOSE') {
+    const options = filepath.startsWith('/') ? {} : { root: '/usr/src/app' };
```

Comment:
maybe?

```suggestion
    const options = {
      ...filepath.startsWith('/') && { root: '/usr/src/app' }
    };
```

Reply:
The ternary is easier to understand imo

Reply:
Fair enough, though why is it hard coded to /usr/src/app?

Reply:
Yeah, probably better to set this with `process.cwd`.

Reply:
cwd? I think it's set somewhere else in an environment variable?

Reply:
The closest thing I can find to an env variable for it is `IMMICH_MEDIA_LOCATION`, which is set to `./upload`.